test/e2e/etcdquorumguard_test.go

```diff
@@ -26,102 +24,74 @@ var pods = make(map[string]podstatus)
 type podinfo map[string]podstatus
 
 func TestEtcdQuorumGuard(t *testing.T) {
-	kclient, err := initialize()
-	if err != nil {
+	cs := framework.NewClientSet("")
+	if err := waitForEtcdQuorumGuardDeployment(cs); err != nil {
 		fmt.Printf("No etcd-quorum-guard deployment present; assume for now it is not configured.\n")
 		return
 		//t.Fatal(err.Error())
```

Comment:
remove

Reply:
When I have all the pieces in place, yes.